packages/terra-consumer-layout/tests/nightwatch/DexLayout.jsx

```diff
@@ -131,16 +132,47 @@ const data = {
   ],
 };
 
-export default () => (
-  <main className="dex-theme">
-    <AppShellExample useRouter>
-      <Layout {...data}>
-        <Route exact path="/" render={() => <h1 style={{ color: 'white' }}>of Thrones more content more content more content more content more content more content more content more content more content more content more content more content more content more content more content more content more content more content more content more content more content more content more content more content more content more content more content more content more content more content more content more content more content more content more content more content more content more content more content more content more content more content more content more content</h1>} />
-        <Route exact path="/inbox" render={() => <h1>You have millions of unread messages</h1>} />
-        <Route exact path="/sent" render={() => <h1>Opps, it is empty</h1>} />
-        <Route exact path="/health" render={() => <h1>Game</h1>} />
-        <Route exact path="/record" render={() => <h1 style={{ color: 'white' }}>of Thrones more content more content more content more content more content more content more content more content more content more content more content more content more content more content more content more content more content more content more content more content more content more content more content more content more content more content more content more content more content more content more content more content more content more content more content more content more content more content more content more content more content more content more content more content</h1>} />
-      </Layout>
-    </AppShellExample>
-  </main>
-);
+export default class DexLayout extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      alerts: ['001', '002'],
+    };
+
+    this.generateAlerts = this.generateAlerts.bind(this);
+    this.dismissAlert = this.dismissAlert.bind(this);
+  }
+
+  generateAlerts() {
+    const alertList = [];
```

Comment:
nit: whenever you find yourself creating an array, then doing a forEach and looping over something to push items on an array, the map function is usually better

Reply:
+1